### ISSUE TITLE
L036 triggers with select targets each on their own line

### ISSUE THREAD

**Issue by tekumara** [sqlfluff/sqlfluff#2050]
## Expected Behaviour
No L036 

## Observed Behaviour
```
L:   1 | P:   1 | L036 | Select targets should be on a new line unless there is
                       | only one select target.
```

## Steps to Reproduce
_test.sql_
```
SELECT 1 AS dv_keyid,
       MIN(t.runid) AS dv_runid,
       MIN(t.dv_distkey) AS dv_distkey,
       t.id
FROM t
```
## Version
sqlfluff, version 0.8.2
python 3.9

## Configuration

Default out-of-the-box config.

**Comment by jpy-git** [comment]
@tekumara this is not a bug. "Select targets should be on a new line unless there is only one select target." includes the first select target.

i.e.
```sql
SELECT
    1 AS dv_keyid,
    MIN(t.runid) AS dv_runid,
    MIN(t.dv_distkey) AS dv_distkey,
    t.id
FROM t
```

(https://docs.sqlfluff.com/en/stable/rules.html#:~:text=class%20Rule_L036(code%2C%20description%2C%20**kwargs))``
